content/about/student_stories.rst

```diff
@@ -9,6 +9,61 @@ Our Student Stories are a special collection of experiences and thoughts from
 the students themselves. Check out what our student employees find exciting and
 worth taking some time to tell you about.
 
+Reflections on my Time at the Open Source Lab, Travis Whitehead
+------------------------------------------------------------------
+
+4/24/19
+
+Back in the Winter of 2017, I came across a job listing for the Open Source Lab
+(OSL) by chance. I had never heard of the OSL, and it certainly did not factor
+in to my decision to pursue computer science at Oregon State University (OSU).
+However, looking back, above all academics the Open Source Lab has been the
+highlight of my computer science experience at OSU.
+
+I'd been running Linux as my "daily driver" since high school, and over time I
+found myself becoming more and more deeply invested in the ideological
+underpinnings of FOSS (Free Open-Source Software). From the security/privacy
+benefits of total transparency into how software functions, to the freedoms of
+being able to learn from, repair, and build upon the software we use every day,
+FOSS offers more for the world than proprietary solutions ever can.
+
+Despite my strong interest in free software, I'd never imagined myself in the
+position of getting paid to contribute to open-source. OSL alumni can tell you
+all about the valuable skills and work experience we pick up at the OSL, but for
+me the biggest thing was to be able to do work that is ethical and important to
+me.
+
+In a world shaped by a for-profit economy, our relationship with software
+and intellectual property is dominated by exclusion. If users cannot afford to
+pay for software, they are excluded access to software, or must access it
```

Comment:
```suggestion
pay for software, they are excluded access to software or must access it
```
Commas separate multiple predicates when there are three or more verb phases, but a [compound predicate](https://www.grammar.com/compound-predicate) with only two verbs phrases requires no punctuation to separate them.